### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added Cart page for Cmart
Screenshot (80)

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  /* padding-left: 10%; */
+}
+.footer {
+  display: flex;
+  bottom: 0;
```
Comment (ZuberDunge) [inline styles/cart.css:23]: what is the use of bottom 0 here? using postitions?

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can merge

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  width: var(--width-100);
+  height: 12vh;
+  background-color: var(--color-navyBlue);
+  gap: var(--size-2);
+}
+.footer-list {
+  display: flex;
+  gap: var(--size-3);
+  color: var(--color-white);
+}
```
Comment (ZuberDunge) [inline styles/cart.css:36]: You can add line space after curly braces.

File: styles/cart.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 10%;
```
Comment (ZuberDunge) [inline styles/cart.css:70]: you can remove this `margin-top: 10%; `as you're using `margin: 10% 0 10% 0;` on next line

File: styles/login.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  left: 21%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  padding-left: 10%;
+}
+.footer {
```
Comment (ZuberDunge) [inline styles/login.css:21]: You've used same styling in other component too, you can just keep one for all pages.

**Review by ZuberDunge — COMMENTED** [review]
Suggested some changes, rest looks good to me. Good to go from my side.

File: styles/cart.css (hunk 1 above)
Comment (dubeyaditya29) [reply styles/cart.css:23]: yeah position is absolute here

File: styles/cart.css (hunk 3 above)
Comment (dubeyaditya29) [reply styles/cart.css:70]: sure will do that

File: styles/login.css (hunk 4 above)
Comment (dubeyaditya29) [reply styles/login.css:21]: Thanks for pointing this out

**Pull request merged**